Application/EdFi.Ods.Common/Models/SuppliedDomainModelProvider.cs

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.Ods.Common.Models.Domain;
+
+namespace EdFi.Ods.Common.Models;
+
+/// <summary>
+/// Implements a simple <see cref="IDomainModelProvider" /> that returns the instance supplied to the constructor.
+/// </summary>
+public class SuppliedDomainModelProvider : IDomainModelProvider
```

Comment:
Added for the sake of CodeGen that now needs to supply the domain model (which is already available in the template context) to the person-related specification instance-based replacements.